core/src/main/resources/hudson/model/View/newJob.properties

```diff
@@ -1,3 +1,9 @@
 NewJob=New {0}
 JobName={0} name
+ItemName.help=Required field
+ItemName.validation.required=This field can not be empty. Please, enter a valid name and press OK button.
+ItemName.validation.invalidFormat=Invalid item name
+ItemType.validation.required=Please, select an item type
```

Comment:
`ItemType.validation.required` is being used [here](https://github.com/jenkinsci/jenkins/pull/2191/files#diff-40f062c03708db68431ba5bb5e48c360R43)